l10n/en/products/vpn/resource-center.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/resource-center
+
+# HTML Page Title
+vpn-resource-center-title = VPN Resource Center
+
+vpn-resource-center-whats-the-deal = What’s the deal with Virtual Private Networks?
+vpn-resource-center-discover-how-they = Discover how they work, why they’re important and what using a VPN can do for you.
+vpn-resource-center-view-all = View all
+vpn-resource-center-mozilla-vpn = { -brand-name-mozilla-vpn }
+vpn-resource-center-start-protecting = Start protecting your identity today
+
+vpn-resource-center-obsessed-with = Obsessed with your privacy since 1998
+vpn-resource-center-get-mozilla-vpn = Get { -brand-name-mozilla-vpn }
+vpn-resource-center-was-this-article = Was this article helpful?
+
+# Variables:
+#   $percent (string) - percentage saved with chosen subscription plan e.g. '40'.
+# Asterisk indicates a footnote for the following string
+vpn-resource-center-on-mozilla-vpn = Save { $percent }% on { -brand-name-mozilla-vpn }*
+# this is used as a footnote for the previous string and should include the asterisk OR matching character for both strings.
+vpn-resource-center-when-you-subscribe = *when you subscribe to a 12-month plan
```

Comment:
Seeing as these two strings are related, it might make sense to move them both into `products/vpn/shared.ftl`?